requirements.txt

```diff
@@ -15,7 +15,7 @@ debtcollector==1.3.0
 decorator==4.0.9
 dj-static==0.0.6
 Django==1.9.5
-django-angular==0.8.2
+git+https://github.com/jrief/django-angular.git@c21d7f013fa5108ba7ecc3c7a33de0251aafc604#egg=django-angular==0.8.3
```

Comment:
Why hashes instead of versions, once again this is a genuine question). If the reason is that we want to use version that is not yet released, please add a comment: "Can be replaced with django-angular==0.8.3 when it is released".